### ISSUE TITLE
Patents database

### ISSUE THREAD

**Issue by ponsfrilus** [epfl-sti/wordpress.theme#97]
Patents database - some thinking on how to integrate patents on the website:

* Where does the information comes from, what the normal "way" of being update of a new patent @dalgetty ?
* How does the patents number works ? Is there a standardized number like `DOI` for paper e.g. WO2017216623 ?
* Is the information from stisrv13 are the same that one can find on [infoscience](https://infoscience.epfl.ch/search?ln=en&cc=Infoscience%2FPatent&p=&f=&rm=&ln=en&sf=&so=d&rg=10&c=Infoscience%2FPatent&c=&of=hb&fct__2=TTO) ?
* Is there a place on the current website where the patents are listed ?
* What's the best way to link a patent to other elements ? Labs ? Prof ? Other ?
* Beside infoscience, is there an official global website / database where one can find relevant information about one patent ? [EspaceNet](https://worldwide.espacenet.com/) is the European Patent Office - but it should be some others...

**Comment by dalgetty** [comment]
Patents in the stisrv13 database are stored as follows:
patents_rel_sti is a relational table between the patents and the profs
patents is the list of patents

In this way several profs can be attributed to the same patent

Until now I inserted them in one go, manually, Excel to MySQL

There is at least one number like WO982734 per patent, but many patents have more than one: for each country in which a patent was registered there is a number. The main number is stored in patents.link, the others in patents.also - this is how it is possible to make a flag and a link for each country (see [https://stisrv13.epfl.ch/test/patents.png])

There is no link between InfoScience and this list of patents. I gathered this info myself in early 2017. It dates from each professor's arrival to that time. To update it manually is the only way I know of keeping it up to date, but it wouldn't take very long

The single best source of info is https://patents.google.com/
